App crashes when Customize notifications is clicked

## Pre-submission checklist
-  I am submitting a bug report, not a feature request.
-  I have searched for similar issues, but did not find any matches.

## Description
When I click the setting "Customize notifications", the app crashes

## Steps to reproduce
1. Go to settings
2. Scroll down
3. Click Customize notifications


## System information
 - Phone: Google Pixel 5
 - Phone Operating System: Android 7.1.1
 - App version: 2.2.0

## Screenshots
If applicable, add screenshots to help explain your problem.
<img width="191" alt="1688885318405" src="https://github.com/iSoron/uhabits/assets/98003627/fb57292a-6431-4143-a8df-489a79accf3d">
<img width="189" alt="1688885345066" src="https://github.com/iSoron/uhabits/assets/98003627/eb567a56-6d6d-4739-b742-e508fec64975">


## Log
```
FATAL EXCEPTION: main
                            Process: org.isoron.uhabits, PID: 11967
                            java.lang.NoClassDefFoundError: Failed resolution of: Landroid/app/NotificationChannel;
                             at org.isoron.uhabits.notifications.AndroidNotificationTray$Companion.createAndroidNotificationChannel(AndroidNotificationTray.kt:171)
                             at org.isoron.uhabits.activities.settings.SettingsFragment.onPreferenceTreeClick(SettingsFragment.kt:103)
                             at androidx.preference.Preference.performClick(Preference.java:1173)
                             at androidx.preference.Preference.performClick(Preference.java:1148)
                             at androidx.preference.Preference$1.onClick(Preference.java:172)
                             at android.view.View.performClick(View.java:5637)
                             at android.view.View$PerformClick.run(View.java:22429)
                             at android.os.Handler.handleCallback(Handler.java:751)
                             at android.os.Handler.dispatchMessage(Handler.java:95)
                             at android.os.Looper.loop(Looper.java:154)
                             at android.app.ActivityThread.main(ActivityThread.java:6119)
                             at java.lang.reflect.Method.invoke(Native Method)
                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
                            Caused by: java.lang.ClassNotFoundException: Didn't find class "android.app.NotificationChannel" on path: DexPathList[[zip file "/data/app/org.isoron.uhabits-1/base.apk"],nativeLibraryDirectories=[/data/app/org.isoron.uhabits-1/lib/x86, /system/lib, /vendor/lib]]
                             at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:56)
                             at java.lang.ClassLoader.loadClass(ClassLoader.java:380)
                             at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
                             at org.isoron.uhabits.notifications.AndroidNotificationTray$Companion.createAndroidNotificationChannel(AndroidNotificationTray.kt:171) 
                             at org.isoron.uhabits.activities.settings.SettingsFragment.onPreferenceTreeClick(SettingsFragment.kt:103) 
                             at androidx.preference.Preference.performClick(Preference.java:1173) 
                             at androidx.preference.Preference.performClick(Preference.java:1148) 
                             at androidx.preference.Preference$1.onClick(Preference.java:172) 
                             at android.view.View.performClick(View.java:5637) 
                             at android.view.View$PerformClick.run(View.java:22429) 
                             at android.os.Handler.handleCallback(Handler.java:751) 
                             at android.os.Handler.dispatchMessage(Handler.java:95) 
                             at android.os.Looper.loop(Looper.java:154) 
                             at android.app.ActivityThread.main(ActivityThread.java:6119) 
                             at java.lang.reflect.Method.invoke(Native Method) 
                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886) 
                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776) 
```

--- a/uhabits-android/build.gradle.kts
+++ b/uhabits-android/build.gradle.kts
@@ -37,7 +37,7 @@ android {
     defaultConfig {
         versionCode = 20200
         versionName = "2.2.0"
-        minSdk = 23
+        minSdk = 28
         targetSdk = 31
         applicationId = "org.isoron.uhabits"
         testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"

